public class DivisionPorCeroException extends Exception{
	public DivisionPorCeroException(String message){
		
	}
}


public static class Main(){
	try{
		int c = divide(1, 0);
	}catch (DivisionPorCeroException e){
		System.out.println("Estas intentando dividir por 0");
	}
}

public static int divide(int a, int b) throws DivisionPorCeroException{
	if(b==0){
		throw new DivisionPorCeroException();
	}
	return a/b
}